tutorial/airfoilopt/multipoint/airfoil_multiopt.py

```diff
@@ -36,7 +36,7 @@
 MP.addProcessorSet("cruise", nMembers=nGroup, memberSizes=nProcPerGroup)
 comm, setComm, setFlags, groupFlags, ptID = MP.createCommunicators()
 outputDirectory = "output"
-if comm.rank == 0:
+if comm.rank == 0 and os.path.exists(outputDirectory) is False:
```

Comment:
This doesn't make a lot of sense to me
- The preferred Python syntax is `and not os.path.exists()`
- What happens if the directory exists? Would this overwrite stuff? I think the preferred solution is to exit with a clear error message, so the user either deletes the old directory or specifies a new directory. Blindly overwriting stuff is not a good idea IMO

Reply:
No, this just ensures that the directory is created only if not there already, it does not overwrite any file